validity-checker/ValidtyChecker.ts

```diff
@@ -6,7 +6,21 @@
 import { GitHubIssue } from '../api/api';
 import { safeLog } from '../common/utils';
 
-const keywords = ['money', 'xbox', 'tiktok', 'tik-tok'];
+const keywords = [
+	'money',
+	'xbox',
+	'tiktok',
+	'tik-tok',
+	'leak',
```

Comment:
nit/not sure if this matters: what if talking about something like `disposable leak` or `leaks in memory` (likely an internal member reported issue)